challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |x| x.capitalize }
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.select { |x| x.downcase == "dog" }
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.select { |x| x.downcase != "dog" }
 end
 
 def count_the_animals(input)
-  #implement your solution here
+  input.length
 end
 
 def fetch_the_first_two(input)
-  #implement your solution here
+  input[0..1]
```

Comment:
`.first` is a bit more readable and is also an enumerable.
https://ruby-doc.org/core-2.5.0/Enumerable.html#method-i-first